app/controllers/profile/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Profile::DashboardController < ApplicationController
```

Comment:
I think we can move the show method here. I am using that @user for the users show page.

Reply:
Is that user show page located at a URI beginning with "/profile"?